roledb.go

```diff
@@ -495,6 +495,26 @@ func (x *RoleGroupCache) InsertGroup(group *AuthGroup) (err error) {
 	return
 }
 
+func LoadOrCreateGroup(icentral *Central, groupName string, createIfNotExist bool) (*AuthGroup, error) {
```

Comment:
This function's signature should be:
func LoadOrCreateGroup(roleDB *RoleGroupDB, groupName string, createIfNotExist bool) (*AuthGroup, error)

This stuff is at a "lower" level that Central, and there shouldn't be any knowledge of Central down here.

Also, it would be best placed at line 95, because it is a function that operates on any object of the interface type RoleGroupDB.